client/src/models/businessConstants.ts

```diff
@@ -0,0 +1,723 @@
+import _ from "lodash";
+
+import { assign_to_dev_helper_namespace } from "src/core/assign_to_dev_helper_namespace";
+import { lang } from "src/core/injected_build_constants";
+
+interface BusinessConceptTextValue {
+  text: string;
+}
+interface BilingualBusinessConceptTextValue {
+  en: string;
+  fr: string;
+}
+
+type RawBusinessConceptInputTextValue =
+  | BilingualBusinessConceptTextValue
+  | BusinessConceptTextValue;
+
+function getNormalizedEntry(
+  obj: RawBusinessConceptInputTextValue
+): BusinessConceptTextValue {
+  if (obj.hasOwnProperty("text")) {
+    return { text: (obj as BusinessConceptTextValue).text };
+  } else if (obj.hasOwnProperty(lang)) {
+    return { text: (obj as BilingualBusinessConceptTextValue)[lang] };
+  } else {
+    throw "shouldn't happen";
```

Comment:
I thought we added a lint for not throwing text. Maybe it only works for JS?

Should probably use the Error class